lib/SQLBuilder.php

```diff
@@ -118,18 +118,22 @@ public function get_where_values(): array
     /**
      * @param list<WhereClause>    $clauses
      * @param array<string,string> $mappedNames
+     * @param array<string>        $columns     Table column names
      *
      * @throws Exception\ExpressionsException
      *
      * @return $this
      */
-    public function where(array $clauses=[], array $mappedNames=[]): static
+    public function where(array $clauses=[], array $mappedNames=[], array $columns=[]): static
     {
+        $escapedNames = $this->connection->determineColumnsThatNeedEscaping($columns);
```

Comment:
Not a big deal, but isn't this a bit overly indirect? Since only `PgsqlAdapter` returns anything other than empty array from `determineColumnsThatNeedEscaping`, why not just get rid of it and instead override `escapeColumns` in `PgsqlAdapter`? Then it's the same work, but with less twistiness.

Reply:
Ok, refactored.